desktop/assets/texts/sk-SK/preferences_dialog.ftl

```diff
@@ -0,0 +1,16 @@
+preferences-dialog = Nastavenia Ruffle
+preference-locked-by-cli = Len na čítanie (nastavené CLI)
+graphics-backend = Grafický backend
+graphics-backend-default = Predvolené
+graphics-power = Nastavenie výkonu
+graphics-power-low = Nízky (napr. iGPU)
+graphics-power-high = Vysoký (napr. GPU)
+language = Jazyk
+audio-output-device = Zariadenie pre výstup zvuku
+audio-output-device-default = Predvolené systémom
+log-filename-pattern = Názov súboru logu
+log-filename-pattern-single-file = Jeden súbor (ruffle.log)
+log-filename-pattern-with-timestamp = S časovou pečiatkou
+storage-backend = Umiestnenie súboru s uloženými pozíciami
```

Comment:
Google translate tells me this means "Location of the saved positions file".

Can you confirm that this is as intended, and correct, @MartySVK? Is this kind of "save" said as "saved position" in Slovak?

Reply:
If we are talking about location for "Save Files" (progress in levels, ...). Yes, it is correct. I am not sure if we have a simpler translation for Save-File.
Also, the same translation is in files for the Czech language.

Reply:
Okay, I see, thank you! ☺️